Assets/MixedRealityToolkit.Services/InputSimulation/ArticulatedHandPoses/ArticulatedHandPose_Poke.json

```diff
@@ -0,0 +1 @@
leMiddleJoint","pose":{"position":{"x":0.005449346732348204,"y":0.0031707696616649629,"z":0.08099328726530075},"rotation":{"x":-0.49786925315856936,"y":0.13922974467277528,"z":-0.07507844269275665,"w":-0.8527824878692627}}},{"joint":"MiddleDistalJoint","pose":{"position":{"x":0.0013555703917518259,"y":-0.01869615726172924,"z":0.09269960224628449},"rotation":{"x":-0.7163864970207214,"y":0.07041004300117493,"z":-0.030646607279777528,"w":-0.6939578652381897}}},{"joint":"MiddleTip","pose":{"position":{"x":0.0004728742642328143,"y":-0.03479576110839844,"z":0.09213778376579285},"rotation":{"x":-0.7163864970207214,"y":0.07041004300117493,"z":-0.030646607279777528,"w":-0.6939578652381897}}},{"joint":"RingMetacarpal","pose":{"position":{"x":0.044932689517736438,"y":-0.04016602039337158,"z":0.013597620651125908},"rotation":{"x":-0.3939853310585022,"y":-0.10114617645740509,"z":0.016117071732878686,"w":0.9133923053741455}}},{"joint":"RingKnuckle","pose":{"position":{"x":0.03491469845175743,"y":-0.003818823955953121,"z":0.047541361302137378},"rotation":{"x":-0.11738020181655884,"y":0.15373656153678895,"z":0.05639626830816269,"w":-0.9795019030570984}}},{"joint":"RingMiddleJoint","pose":{"position":{"x":0.023768775165081025,"y":-0.01135534793138504,"z":0.08033758401870728},"rotation":{"x":-0.7923092842102051,"y":0.16401034593582154,"z":-0.02978098951280117,"w":-0.5869977474212647}}},{"joint":"RingDistalJoint","pose":{"position":{"x":0.02067880891263485,"y":-0.031320542097091678,"z":0.0737735852599144},"rotation":{"x":-0.9346709847450256,"y":0.0874316394329071,"z":-0.023773543536663057,"w":-0.344605952501297}}},{"joint":"RingTip","pose":{"position":{"x":0.020386409014463426,"y":-0.04289411008358002,"z":0.06018315628170967},"rotation":{"x":-0.9346709847450256,"y":0.0874316394329071,"z":-0.023773543536663057,"w":-0.344605952501297}}},{"joint":"PinkyMetacarpal","pose":{"position":{"x":0.05288681760430336,"y":-0.041848354041576388,"z":0.01654883660376072},"rotation":{"x":-0.33144858479499819,"y":0.002071807160973549,"z":0.085218146443367,"w":0.9396145343780518}}},{"joint":"PinkyKnuckle","pose":{"position":{"x":0.050300415605306628,"y":-0.011202438734471798,"z":0.054917603731155398},"rotation":{"x":-0.16419324278831483,"y":0.1696346402168274,"z":0.12252454459667206,"w":-0.9639865159988403}}},{"joint":"PinkyMiddleJoint","pose":{"position":{"x":0.04166591167449951,"y":-0.017666997388005258,"z":0.07580538094043732},"rotation":{"x":-0.7474591135978699,"y":0.20672142505645753,"z":0.04626481607556343,"w":-0.6297129392623901}}},{"joint":"PinkyDistalJoint","pose":{"position":{"x":0.03587989881634712,"y":-0.03386271744966507,"z":0.0722469910979271},"rotation":{"x":-0.928327202796936,"y":0.13445810973644257,"z":0.1272566169500351,"w":-0.3232197165489197}}},{"joint":"PinkyTip","pose":{"position":{"x":0.03135494887828827,"y":-0.04178089275956154,"z":0.06164591759443283},"rotation":{"x":-0.928327202796936,"y":0.13445810973644257,"z":0.1272566169500351,"w":-0.3232197165489197}}}]}
```

Comment:
What is the frame of reference here? Relative to world origin 0,0,0 using camera.forward? Or, is it relative to the camera?

Reply:
Joint poses in ArticulatedHandPose are stored

- in camera space
- for the right hand

For the left hand they simply get mirrored. I added comments on the ArticulatedHandPose class yesterday to clarify, let me know if i should add something there.